modules/ocf_mirrors/manifests/projects/devuan.pp

```diff
@@ -0,0 +1,20 @@
+class ocf_mirrors::projects::devuan {
+  file {
+    '/opt/mirrors/project/devuan':
+      ensure  => directory,
+      source  => 'puppet:///modules/ocf_mirrors/project/devuan',
+      owner   => mirrors,
+      group   => mirrors,
+      mode    => '0755',
+      recurse => true;
+  }
+
+# Looks like devuan doesn't have a timestamp file, so omitted ocf_mirrors::monitoring
+
+  ocf_mirrors::timer { 'devuan':
+    exec_start => '/opt/mirrors/project/devuan/sync-archive',
+    hour       => '*',
```

Comment:
we can probably do `0/6` for the as they recommend in https://files.devuan.org/MIRRORS.txt